pkg/openapiclient/local_crds.go

```diff
@@ -33,44 +32,42 @@ var metadataSchemas map[string]*spec.Schema = func() map[string]*spec.Schema {
 
 // client which provides openapi read from files on disk
 type localCRDsClient struct {
-	fs   fs.FS
-	dirs []string
+	fileSystems []fs.FS
 }
 
 // Dir should have openapi files following directory layout:
 // myCRD.yaml (groupversions read from file)
-func NewLocalCRDFiles(fs fs.FS, dirPaths []string) openapi.Client {
+func NewLocalCRDFiles(fs ...fs.FS) openapi.Client {
```

Comment:
Yup, what do you mean ?

Reply:
I was wondering if this changed functionality to not allow using a relative dirpath from an embed.FS; just noting that it doesnt matter since we can make a subtree fs from any fs